sdk/src/transaction/sanitized.rs

```diff
@@ -58,6 +58,21 @@ impl From<Hash> for MessageHash {
 }
 
 impl SanitizedTransaction {
+    /// Create a sanitized transaction
+    pub fn new(
+        tx: SanitizedVersionedTransaction,
+        message: SanitizedMessage,
+        message_hash: Hash,
+        is_simple_vote_tx: bool,
+    ) -> Self {
+        Self {
+            message,
+            message_hash,
+            is_simple_vote_tx,
+            signatures: tx.signatures,
+        }
+    }
+
```

Comment:
Not feeling great about this function, one can cobble together totally unrelated `tx` and `message` into a sanitized_transaction that ain't sanitized at all.